<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeathNotice</class>
 <widget class="QWidget" name="DeathNotice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; padding: 5px; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="possessRandomWombat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QLabel" name="info1">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MoolBoran</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="openBurrowList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="info2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>50</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white;</string>
   </property>
   <property name="text">
    <string>died of old age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info3">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white;</string>
   </property>
   <property name="text">
    <string>Possess new wombat:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deathNoticeClose">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>info1</zorder>
  <zorder>info2</zorder>
  <zorder>info3</zorder>
  <zorder>deathNoticeClose</zorder>
  <zorder>quitButton</zorder>
  <zorder>okButton</zorder>
  <zorder>openBurrowList</zorder>
  <zorder>possessRandomWombat</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
